### DISCUSSION TITLE
github is not a social media site.

### DISCUSSION THREAD

**Discussion by mechabubba** [community/community#67237] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

i do not want to see any recommended repositories on my home feed. i will not have any use for them as a developer. i use this site to publish my code for people to use if they *search for it.* i'm not going to use deepObjectEquals because it is suggested to me on my home feed - im assuming its there because i worked on a javascript project recently. i will use it if i need it (which i wont), not because its suggested.

use your heads.

**Comment by erdajt** [comment]
Strongly disagree with your take for a large number of reasons which I think are not even worth stating for an argument like yours...

**Reply by BDARmech** [reply]
I think this would be an ideal social platform for some like minded people.
